package.json

```diff
@@ -62,23 +56,30 @@
     "semantic-ui-css": "2.4.1",
     "semantic-ui-react": "0.88.0",
     "sjs-base-model": "1.9.0",
-    "ts-optchain": "0.1.8",
-    "typescript": "3.6.2",
     "uuid": "3.3.3"
   },
   "devDependencies": {
     "@craco/craco": "5.4.0",
     "@types/history": "4.7.3",
+    "@types/jest": "24.0.18",
+    "@types/node": "12.7.3",
+    "@types/react": "16.9.2",
+    "@types/react-dom": "16.9.0",
     "@types/react-redux": "7.1.2",
     "@types/react-router-dom": "4.3.5",
     "@types/uuid": "3.4.5",
     "cross-env": "5.2.1",
     "generate-template-files": "2.2.0",
     "husky": "3.0.5",
-    "lint": "0.7.0",
+    "node-sass": "4.12.0",
+    "prettier": "1.18.2",
     "pretty-quick": "1.11.1",
+    "react-scripts": "3.1.1",
+    "ts-optchain": "0.1.8",
```

Comment:
Shouldn't `ts-optchain` be in dependencies?

Reply:
yeah let me fix that